paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetScreenshotTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.stripe.android.customersheet
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.stripe.android.customersheet.ui.CustomerSheetScreen
+import com.stripe.android.utils.screenshots.FontSize
+import com.stripe.android.utils.screenshots.PaparazziRule
+import com.stripe.android.utils.screenshots.PaymentSheetAppearance
+import com.stripe.android.utils.screenshots.SystemAppearance
+import org.junit.Rule
+import org.junit.Test
+
+class CustomerSheetScreenshotTest {
+    @get:Rule
+    val paparazzi = PaparazziRule(
+        SystemAppearance.values(),
+        FontSize.values(),
+        PaymentSheetAppearance.values(),
```

Comment:
I think we determined that we only want to test all variants in component screenshot test. For tests that cover the entire screen, we’re fine with only testing one specific variant.